app/schema/ddi-index-api/persons/get.js

```diff
@@ -2,15 +2,20 @@ const Joi = require('joi')
 const { stripTimeFromUTC } = require('../../../lib/date-helpers')
 
 const personsFilter = Joi.object({
-  firstName: Joi.string().required(),
-  lastName: Joi.string().required(),
+  firstName: Joi.string().optional(),
+  lastName: Joi.string().optional(),
+  orphaned: Joi.boolean().optional(),
   dateOfBirth: Joi.date().iso().empty(null).allow(null).optional().custom((value) => {
     return stripTimeFromUTC(new Date(value))
   }),
   dobDay: Joi.number().optional().allow('').strip(),
   dobMonth: Joi.number().optional().allow('').strip(),
-  dobYear: Joi.number().optional().allow('').strip()
-})
+  dobYear: Joi.number().optional().allow('').strip(),
+  sortKey: Joi.string().allow('owner').optional(),
```

Comment:
Does this not need to allow other column values too, e.g. dateOfBirth ?

Reply:
api isn't implemented yet.